backend/employee-payroll-service/src/main/java/com/mycompany/employee/payroll/service/exception/DataEmptyException.java

```diff
@@ -0,0 +1,9 @@
+package com.mycompany.employee.payroll.service.exception;
+
+public class DataEmptyException extends BaseServiceException {
```

Comment:
I think it's better to use data not found unless I misunderstand what you aim to.